packages/disputer/package.json

```diff
@@ -6,6 +6,8 @@
     "@uma/common": "^2.6.0",
     "@uma/core": "^2.7.0",
     "@uma/financial-templates-lib": "^2.6.1",
+    "@uma/contracts-node": "^0.1.0",
+    "@uma/contracts-node-0-1-0": "npm:@uma/contracts-node@0.1.0",
```

Comment:
why do we need this import?

Reply:
When we dynamically look up the abi/bytecode of specific versions, our method is to create an alias that has that version number in its package name. So 2.0.1 maps to contracts-node@0.1.0, which we alias to contracts-node-0-1-0. 

See this function for details: https://github.com/UMAprotocol/protocol/blob/06094f2b2c9defbcc332da4de64e2bf1a6ed91fa/packages/common/src/MultiVersionTestHelpers.ts#L25-L34.